### PR TITLE
DOCS-2282 Risk score section rewritten

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/api-discovery/risk-score.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 [API Discovery](overview.md) automatically calculates a **risk score** for each endpoint in your API inventory. The risk score allows you to understand which endpoints are most likely to be an attack target and therefore should be the focus of your security efforts.
 
+## Overviewing dashboard
```
Comment (AnastasiaTWW) [inline docs/latest/api-discovery/risk-score.md:5]: again, it seems that it is enough to have this decription oin the dashboard article and 1 sentence here pointint gusers to the dash article

File: docs/latest/api-discovery/risk-score.md [hunk 2]
```diff
@@ -28,7 +37,7 @@ Risk score may be from `1` (lowest) to `10` (highest):
 * `1` means no risk factors for this endpoint.
 * Risk score is not displayed (`N/A`) for the unused endpoints.
 * Sort by risk score in the **Risk** column.
-* Filter out `High`, `Medium` or `Low` using the **Risk score** filter.
+* Filter `High`, `Medium` or `Low` using the **Risk score** filter.
 
 To understand what caused the risk score for the endpoint and how to reduce the risk, go to the endpoint details:
```
Comment (AnastasiaTWW) [inline docs/latest/api-discovery/risk-score.md:43]: the comment for "## Customizing risk score calculation"

let's move the default state description to Risc score factors - you can event combine everything into 1 table

Comment (AnastasiaTWW) [reply docs/latest/api-discovery/risk-score.md:43]: and use the "## Customizing risk score calculation" only for its purpose

File: docs/latest/api-discovery/risk-score.md (hunk 1 above)
Comment (MaximBashurov) [reply docs/latest/api-discovery/risk-score.md:5]: And again disagree. The reason is the same - overviewing the dashboard is a valuable scenario. Too cool to hide. :-)

File: docs/latest/api-discovery/risk-score.md [hunk 3]
```diff
@@ -4,18 +4,20 @@
 
 ## Risc score factors
 
-The risk score is made up of various factors, including:
+The risk score is made up of various factors, each having its weight when calculating the final risk score. By default, the highest weight from all criteria is used as endpoint risk score.
```
Comment (AnastasiaTWW) [inline docs/latest/api-discovery/risk-score.md:7]: or the average values of all weights is a final weight?

File: docs/latest/api-discovery/risk-score.md [hunk 4]
```diff
@@ -4,18 +4,20 @@
...
-* A **large number of parameters** increasing the number of attack directions.
-* **XML or JSON objects** passed in the endpoint request may be used by the attackers to transfer malicious XML external entities and injections to the server.
+| Factor | Description | Default weight |
+| --- | --- | --- |
+| Active vulnerabilities | [Active vulnerabilities](../about-wallarm/detecting-vulnerabilities.md)  may result in unauthorized data access or corruption. | 9 |
+| Potentially vulnerable to BOLA | Presence of the [variable path parts](exploring.md#variability-in-endpoints), such as user IDs, e.g. `/api/articles/author/{parameter_X}`. Attackers can manipulate object IDs and, in case of insufficient request authentication, either read or modify the object sensitive data ([BOLA attacks](../admin-en/configuration-guides/protecting-against-bola.md)). | 6 |
+| Parameters with sensitive data | Rather than directly attacking APIs, attackers can steal [sensitive data](overview.md#sensitive-data-detection) and use it to seamlessly reach your resources. | 8 |
+| Number of query and body parameters | A large number of parameters increasing the number of attack directions. | 6 |
```
Comment (AnastasiaTWW) [inline docs/latest/api-discovery/risk-score.md:14]: A large number of parameters increase the number of attack directions.

File: docs/latest/api-discovery/risk-score.md [hunk 5]
```diff
@@ -4,18 +4,20 @@
...
+| Accepts XML / JSON objects | XML or JSON objects passed in the endpoint request may be used by the attackers to transfer malicious XML external entities and injections to the server. | 6 |
```
Comment (AnastasiaTWW) [inline docs/latest/api-discovery/risk-score.md:15]: XML or JSON objects passed in requests may be used by attackers to transfer malicious XML external entities and injections to the server.

File: docs/latest/api-discovery/risk-score.md [hunk 6]
```diff
@@ -28,33 +30,22 @@ Risk score may be from `1` (lowest) to `10` (highest):
...
-    | Accepts XML / JSON objects | 6 |
-    | Allows uploading files to the server | 6 |
-
 To change how risk score is calculated: 
 
 1. Click the **Configure API Discovery** button in the **API Discovery** section.
+1. Switch to the **Risk scoring** tab.
 1. Select calculation method: highest or average weight.
 1. If necessary, disable factors you do not want to affect a risk score.
 1. Set weight for the remaining.
```
Comment (AnastasiaTWW) [inline docs/latest/api-discovery/risk-score.md:51]: weightS

**Pull request merged**